Resources/doc/reference/action_create_edit.rst

```diff
@@ -14,26 +14,132 @@ of the fields and forms available in these views and any other relevant settings
 Basic configuration
 -------------------
 
+SonataAdmin Options that may affect the create or edit view:
+
+.. code-block:: yaml
+
+    sonata_admin:
+        options:
+            html5_validate:       true     # enable or disable html5 form validation
+            confirm_exit:         true     # enable or disable a confirmation before navigating away
+            use_select2:          true     # enable or disable usage of the Select2 jQuery library (https://github.com/select2/select2)
```

Comment:
Can you please wrap the long comments here? They make an horizontal scroll appear.